pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -66,22 +115,23 @@ def set_infectiousness(person: Person):
         ----------
         Person : Person
             Instance of person class with infection status attributes
-
-        Returns
-        -------
-        float
-            Infectiousness of a person
+        time : float
+            Current simulation time
 
         """
         init_infectiousness = np.random.gamma(1, 1)
         if person.infection_status == InfectionStatus.InfectASympt:
-            infectiousness = (init_infectiousness
-                              * pe.Parameters.instance().asympt_infectiousness)
+            infectiousness = init_infectiousness *\
+                             pe.Parameters.instance().asympt_infectiousness
```

Comment:
```suggestion
            infectiousness = (init_infectiousness *
                              pe.Parameters.instance().asympt_infectiousness)
```
See [this](https://github.com/SABS-R3-Epidemiology/epiabm/pull/92#pullrequestreview-905912641) comment to Luke about backslashes - N.B this needs changing in multiple places

Reply:
I removed those I inserted that could be done without breaking flake 8.